ok
'/groups/gaa50073/park-yuna/datasets/VOCtest_06-Nov-2007.tar' -> '/local/42638261.1.gpu/datasets/VOCtest_06-Nov-2007.tar'
'/groups/gaa50073/park-yuna/datasets/VOCtrainval_11-May-2012.tar' -> '/local/42638261.1.gpu/datasets/VOCtrainval_11-May-2012.tar'
VOCtest_06-Nov-2007.tar
VOCtrainval_11-May-2012.tar
VOCdevkit
VOCtest_06-Nov-2007.tar
VOCtrainval_11-May-2012.tar
VOC2007
VOC2012
Already on 'feat-voc-test'
M	src/scripts/abci/test.sh
Your branch is up to date with 'origin/feat-voc-test'.
/home/aaf15782nj/anaconda3/envs/new/lib/python3.11/site-packages/torchvision/io/image.py:13: UserWarning: Failed to load image Python extension: '/home/aaf15782nj/anaconda3/envs/new/lib/python3.11/site-packages/torchvision/image.so: undefined symbol: _ZN5torch3jit17parseSchemaOrNameERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE'If you don't plan on using image functionality from `torchvision.io`, you can ignore this warning. Otherwise, there might be something wrong with your environment. Did you have `libjpeg` or `libpng` installed before building `torchvision` from source?
  warn(
['main.py', 'voc_abci', 'default.dataset_dir=/local/42638261.1.gpu/datasets/', 'default.seed=100', 'augment.name=[nan]']
nan
MAKE DIR /groups/3/gaa50073/park-yuna/segmentation/output/seed100/voc_abci/Nan/
CUDA is available: True
using device: cuda:0
load data from : /local/42638261.1.gpu/datasets/VOCdevkit/VOC2007/
train dataset len : 1464
val dataset len : 1449
test dataset len : 210
Epoch 1/175 [Train]:   0%|          | 0/1464 [00:00<?, ?it/s]